slick.core.js

```diff
@@ -4,23 +4,33 @@
  * @namespace Slick
  */
 
-(function ($) {
+(function (window) {
   /***
    * An event object for passing data to event handlers and letting them control propagation.
    * <p>This is pretty much identical to how W3C and jQuery implement events.</p>
    * @class EventData
    * @constructor
    */
-  function EventData() {
+  function EventData(event, args) {
+
+    var nativeEvent = event;
+    var arguments_ = args;
     var isPropagationStopped = false;
     var isImmediatePropagationStopped = false;
+    var isDefaultPrevented = false;
+    var returnValues = [];
+    var returnValue = undefined;
+
+    this.target = nativeEvent ? nativeEvent.target : undefined;
 
     /***
      * Stops event from propagating up the DOM tree.
      * @method stopPropagation
      */
     this.stopPropagation = function () {
       isPropagationStopped = true;
+      if(nativeEvent)
+        nativeEvent.stopPropagation();
```

Comment:
could you please wrap the code in curly brace to avoid future bug if someone adds a line for whatever reason